src/convertMswMatchToPact.ts

```diff
@@ -24,7 +24,7 @@ export const convertMswMatchToPact =  ({
           headers: headers?.excludeHeaders
             ? omit(match.request.headers['_headers'], headers.excludeHeaders)
             : match.request.headers['_headers'],
-          body: match.request.bodyUsed ? match.request.body : undefined
+          body: match.request.body ? match.request.body : undefined
```

Comment:
```suggestion
          body: match.request.body
```
is enough in this case. If `body` can be either string or undefined, so if body is falsy then it must be undefined

![image](https://user-images.githubusercontent.com/11528830/167089187-9140e205-54a8-495b-8b03-d45cc8935a7c.png)

Reply:
Without this explicit check, the request body is written to the pact file as "" when there is no body. This may cause different behaviour in pact (using vanilla pact, there is no `request.body` attribute for requests with no body). I've made the change you suggested on this branch: https://github.com/simonn01/pact-msw-adapter/tree/test/fix/request_body -- if you run tests you'll see the assertion that no request body is written to the pact file fails for the GET request.
If we want to remove the check here, we need to evaluate the body to `undefined` somewhere earlier in the code.

Reply:
We could change it to `match.request.body ?? undefined`

Reply:
Rather `||` because we get a falsy value on the left hand (`""`) side but want `undefined` instead.

Reply:
Good shout, we do want `undefined` explicitly